zproject/prod_settings_template.py

```diff
@@ -409,9 +409,10 @@
 #S3_REGION = None
 #S3_ENDPOINT_URL = None
 
-# Maximum allowed size of uploaded files, in megabytes.  DO NOT SET
-# ABOVE 80MB.  The file upload implementation doesn't support chunked
-# uploads, so browsers will crash if you try uploading larger files.
```

Comment:
I think the reason here is incorrect. The main reason why large file uploads fail is that we keep the entire file content in server memory.
https://github.com/zulip/zulip/blob/master/zerver/lib/upload.py#L913

I think the browsers should be pretty smart enough to load only the necessary file content in memory at a time during POST upload requests. So the limitation here is mostly in the server.

https://github.com/zulip/zulip/pull/16716 should fix this limitation.

Reply:
Hm, interesting!  I was going based on the comments on #276.

@Sourabh025, care to do a quick follow-up to update this comment -- though it sounds like this limitation may be short-lived?

Reply:
Sure @alexmv I will apply these changes.

Reply:
@alexmv Yes. This should be in Zulip 4.